app/models/user.rb

```diff
@@ -81,24 +81,38 @@ class User < ApplicationRecord
   validates :locale, inclusion: {
     in: Rails.application.config.i18n.available_locales.map(&:to_s).push(nil)
   }
-  validate :password_complexity
+
+  # add unless statement here instead of the concern for readability
+  # TODO: refactor the Password validator concern
```

Comment:
Will this be done in another PR? :) Feel free to create an issue for this as well just to document.

Reply:
This is one of the setbacks on making this PR short. I agree on having this as another ticket.